docs/en/setup/backend/backend-storage.md

```diff
@@ -54,6 +56,34 @@ storage:
     concurrentRequests: ${SW_STORAGE_ES_CONCURRENT_REQUESTS:2} # the number of concurrent requests
 ```
 
+### ElasticSearch 6 With Https SSL Encrypting communications.
+
+example: 
+
+```yaml
+storage:
+  elasticsearch:
+    # nameSpace: ${SW_NAMESPACE:""}
+    user: ${SW_ES_USER:""} # User needs to be set when Http Basic authentication is enabled
+    password: ${SW_ES_PASSWORD:""} # Password to be set when Http Basic authentication is enabled
+    clusterNodes: ${SW_STORAGE_ES_CLUSTER_NODES:localhost:443}
+    keyStorePath: ${SW_SW_STORAGE_ES_SSL_JKS_PATH:"../es_keystore.jks"}
+    keyStorePass: ${SW_SW_STORAGE_ES_SSL_JKS_PASS:"dangerous"}
```

Comment:
seems elasticsearch doesn't support Keystore password according to https://www.elastic.co/guide/en/elasticsearch/reference/6.8/secure-settings.html. But this feature may be added in the future, I agree with your current design, well, just to pick an empty default `password` value would be a better choice though

Reply:
Why mark this `resolved`?

Reply:
not refresh my page, ignore this.